lib/xlsx/xform/pivot-table/pivot-table-xform.js

```diff
@@ -0,0 +1,191 @@
+      <pivotFields count="${cacheFields.length}">
+        ${renderPivotFields(model)}
+      </pivotFields>
+      <rowFields count="${rows.length}">
+        ${rows.map(rowIndex => `<field x="${rowIndex}" />`).join('\n    ')}
+      </rowFields>
+      <rowItems count="1">
+        <i t="grand"><x /></i>
+      </rowItems>
+      <colFields count="${columns.length}">
+        ${columns.map(columnIndex => `<field x="${columnIndex}" />`).join('\n    ')}
+      </colFields>
+      <colItems count="1">
+        <i t="grand"><x /></i>
+      </colItems>
+      <dataFields count="${values.length}">
+        <dataField
+          name="Sum of ${cacheFields[values[0]].name}"
+          fld="${values[0]}"
+          baseField="0"
+          baseItem="0"
+        />
+      </dataFields>
+      <pivotTableStyleInfo
+        name="PivotStyleLight16"
+        showRowHeaders="1"
+        showColHeaders="1"
+        showRowStripes="0"
+        showColStripes="0"
+        showLastColumn="1"
+      />
+      <extLst>
+        <ext
+          uri="{962EF5D1-5CA2-4c93-8EF4-DBF5C05439D2}"
+          xmlns:x14="http://schemas.microsoft.com/office/spreadsheetml/2009/9/main"
+        >
+          <x14:pivotTableDefinition
+            hideValuesRow="1"
+            xmlns:xm="http://schemas.microsoft.com/office/excel/2006/main"
+          />
+        </ext>
+        <ext
+          uri="{747A6164-185A-40DC-8AA5-F01512510D54}"
+          xmlns:xpdl="http://schemas.microsoft.com/office/spreadsheetml/2016/pivotdefaultlayout"
+        >
+          <xpdl:pivotTableDefinition16
+            EnabledSubtotalsDefault="0"
+            SubtotalsOnTopDefault="0"
+          />
+        </ext>
+      </extLst>
+    `);
+
+    xmlStream.closeNode();
+  }
+
+  parseOpen(node) {
+    // TK
+  }
+
+  parseText(text) {
+    // TK
+  }
+
+  parseClose(name) {
+    // TK
+  }
+
+  reconcile(model, options) {
+    // TK
+  }
+}
+
+// Helpers
+
+function renderPivotFields(pivotTable) {
+  /* eslint-disable no-nested-ternary */
+  return pivotTable.cacheFields
+    .map((cacheField, fieldIndex) => {
+      const fieldType =
+        pivotTable.rows.indexOf(fieldIndex) >= 0
+          ? 'row'
+          : pivotTable.columns.indexOf(fieldIndex) >= 0
+          ? 'column'
+          : pivotTable.values.indexOf(fieldIndex) >= 0
+          ? 'value'
+          : null;
+      return renderPivotField(fieldType, cacheField.sharedItems);
+    })
+    .join('');
+}
+
+function renderPivotField(fieldType, sharedItems) {
+  // fieldType: 'row', 'column', 'value', null
+
+  const defaultAttributes = 'compact="0" outline="0" showAll="0" defaultSubtotal="0"';
+
+  if (fieldType === 'row' || fieldType === 'column') {
+    const axis = fieldType === 'row' ? 'axisRow' : 'axisCol';
+    return `
+      <pivotField axis="${axis}" ${defaultAttributes}>
+        <items count="${sharedItems.length + 1}">
+          ${[...range(0, sharedItems.length)].map(index => `<item x="${index}" />`).join('\n              ')}
```

Comment:
You may try this way:
```suggestion
          ${sharedItems.map((item, index) => `<item x="${index}" />`).join('\n              ')}
```

Reply:
Clever. I like this. I will update and force push this.